.githooks/pre-commit

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+#set -x
+PREFIX="pre-commit:"
+RESULTS_FILE=tfsec-scan-output.txt
+TFSEC_RESULTS_DIR=/tmp/tfsec-results
+TFSEC_WORKING_DIR=/tmp/tfsec-files
+
+initialise() {
+    mkdir -p $TFSEC_WORKING_DIR
+    mkdir -p $TFSEC_RESULTS_DIR
+    rm  $TFSEC_WORKING_DIR/* 2> /dev/null
```

Comment:
Remove stray space.

Why are you redirecting STDERR to /dev/null?

Reply:
ok. Redirecting as it gives an error if this is run for the first time

Reply:
Righto, maybe you could:

- see if the directory exists
- if yes, delete it & recreate it
- if no, create it